.project

```diff
@@ -19,8 +19,8 @@
 			<name>com.redhat.ceylon.eclipse.ui.ceylonBuilder</name>
 			<arguments>
 				<dictionary>
-					<key>compileJs</key>
```

Comment:
I didn't change it intentionally. Maybe has to do with the fact that it's a PR against 1.2.0-fixes?